writer_test.go

```diff
@@ -83,6 +87,30 @@ func testWriterClose(t *testing.T) {
 	}
 }
 
+func testWriterRequiredAcksNone(t *testing.T) {
+	topic := makeTopic()
+	createTopic(t, topic, 1)
+	defer deleteTopic(t, topic)
+
+	w := &Writer{
+		Addr:     TCP("localhost:9092"),
+		Topic:    topic,
+		Balancer: &RoundRobin{},
+		RequiredAcks: RequireNone,
+	}
+	defer w.Close()
+
+	msg := Message{
+		Key:   []byte("ThisIsAKey"),
+		Value: []byte("Test message for required acks test")}
+
+	err := w.WriteMessages(context.Background(), msg)
+	if err != nil {
+		t.Error(err)
```

Comment:
Can we switch this to `t.Fatal(err)` rather than following up with `t.FailNow()` since that's the intent anyways?

Reply:
Sure